internal/cli/commands.go

```diff
@@ -305,6 +321,87 @@ func (c *baseCommand) flagSet(bit flagSetBit, f func(*flag.Sets)) *flag.Sets {
 		})
 	}
 
+	if bit&flagSetNomadClient != 0 {
+		f := set.NewSet("Nomad Cluster Options")
+		f.StringVar(&flag.StringVar{
+			Name:    "address",
```

Comment:
I elected to maintain parity between the Nomad CLI and Nomad Pack for connection flags so that users familiar with one can carry that over to the other.